README.md

```diff
@@ -1,2 +1,75 @@
-# oam-runtime
-A set of libraries for building OAM runtimes
+# OAM Kubernetes Project
+
+The OAM Kubernetes runtime project is a collection of Golang helper libraries and OAM api type
+ definitions. 
+
+It is designed to help OAM platform builders rather than being used directly by developers
```

Comment:
These two paragraphs are verbose and repeating. 

Could you compress the main points in one paragraph (i.e. this repo is platform builder facing and important for ensuring interoperability in OAM platforms)?

Reply:
I actually expended a bit here to make our point loud and clear.

Reply:
That should be the reason I feel it's verbose :-) Personally I'd prefer one sentence/paragraph to tell out the main points.

Reply:
What is the difference between an "OAM platform builder" and a "developer"? Does developer mean "folks developing OAM", or is it synonymous with "end-user" in this context?

Reply:
a developer is an OAM application developer, aka "end-user".  Do you prefer to the term "application developer" or "end-user"?